rotkehlchen/api/v1/fields.py

```diff
@@ -482,13 +482,13 @@ class AssetField(fields.Field):
     def __init__(
             self,
             *,
-            expected_type: Union[
-                type[Asset],
-                type[AssetWithNameAndType],
-                type[AssetWithOracles],
-                type[CryptoAsset],
-                type[EvmToken],
-            ],
+            expected_type: type[Union[
+                Asset,
+                AssetWithNameAndType,
+                AssetWithOracles,
+                CryptoAsset,
+                EvmToken,
+            ]],
```

Comment:
in python 3.10 we can use `|` for the types. Don't know if it solves the issue of having to add `type`

Reply:
no it doesn't. Would just switch out the union and replace with `X | Y | Z`